index.html

```diff
@@ -3,9 +3,85 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Brewery Finder</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/bulma.sass">
+    <link rel="stylesheet" href= "https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.css">
+    <script src="https://use.fontawesome.com/releases/v5.14.0/js/all.js"></script>
+</head>
 </head>
 <body>
+    <header>
+        <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false">
+            <span aria-hidden="true"></span>
+            <span aria-hidden="true"></span>
+            <span aria-hidden="true"></span>
+          </a>
+        <section class="hero is-medium is-primary is-bold">
+            <div class="hero-body">
+              <div class="container">
+                <h1 class="title">
+                  A Hop, Skip, and a Jump
+                </h1>
+                <h2 class="subtitle">
+                  ...away from that best breweries in town.
+                </h2>
+              </div>
+            </div>
+          </section>
+    </header>
     
 </body>
```

Comment:
Careful here. You have content after closing off the body.

Reply:
Good catch!

Reply:
corrected.